uportal-war/src/main/java/org/apereo/portal/portlets/backgroundpreference/BackgroundSetSelectionStrategy.java

```diff
@@ -33,6 +33,9 @@
 
     String[] getImageThumbnailSet(PortletRequest req);
 
+    @since 4.3.1
```

Comment:
Have the other ADA fixes been targeting release 4.3.2? If so, I don't believe this would be a candidate for backporting.